index.html

```diff
@@ -1,23 +1,91 @@
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
     <title>Document</title>
-</head>
-<body>
-    <section class="header-nav">
-        <div class="container">
-            <img src="images/↳ Time.png" alt="time">
-            <img src="images/vec.svg" alt="time" id="capacity">
-            <img src="images/Cap.png" alt="time" id="cap">
-
-
+  </head>
+  <header>
+    <nav class="status-bar">
+      <p class="hour">9:41</p>
+      <div class="icons">
+        <div class="network">
+          <img src="images/net.svg" alt="connection" />
         </div>
-        
+        <div class="wifi">
+          <img src="images/wifi.svg" alt="wifi icon" />
+        </div>
+        <div class="battery">
+          <img src="images/vec.svg" alt="battery icon" />
+        </div>
+      </div>
+      </nav>
+    <nav class="menu">
+      <h6 class="name">Kiprotich Kimutai</h6>
+      <button
+        type="button"
+        aria-label="open close main menu"
+        class="toolbar-icon"
+      >
+        <img src="images/menu.png" alt="" />
+      </button>
+    </nav>
+  </header>
+  <main>
+    <section class="content">
+      <h1 class="title">Hey There <br />I’m Kimutai</h1>
+      <h3 class="subtitle">
+        I’am a Software Developer
+      </h3>
+      <p class="description">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social-media">
+        <ul>
+        <li><a href="https://github.com/Kimutai01">
+          <img
+            class="social-media-icon"
+            src="images/social/github.png"
+            alt="github"
+          />
+        </a></li>
+        <li><a href="https://www.linkedin.com/in/kimutai-kiprotich-1b5045216/">
+          <img
+            class="social-media-icon"
+            src="images/social/linkedin.png"
+            alt="linkedin"
+          />
+        </a></li>
+       <li> <a href="#">
+        <img
+          class="social-media-icon"
+          src="images/social/angelist.png"
+          alt="angelist"
+        />
+      </a></li>
+        <li><a href="https://twitter.com/Kimutai_01?t=oB6Cj93u_SKHZdHeoomTpg&s=09">
+          <img
+            class="social-media-icon"
+            src="images/social/twitter.png"
+            alt="twitter"
+          />
+        </a></li>
+        <li><a href="#">
+          <img
+            class="social-media-icon"
+            src="images/social/medium.png"
+            alt="medium"
+          />
+        </a></li>
+      </ul>
+      </div>
+      <div class="illustration">
+        <img src="images/illustration/illustration-header-mobile.png" alt="illustrations" />
```

Comment:
- [ ] [OPTIONAL] For background images, please ensure to use proper background images(in your stylesheet) instead of img tags. 🤗